apps/wing/src/cli.ts

```diff
@@ -168,7 +168,7 @@ async function main() {
     )
     .option("-r, --rootId <rootId>", "App root id")
     .option("-v, --value <value>", "Platform-specific value in the form KEY=VALUE", addValue, [])
-    .option("--values <file>", "Yaml file with Platform-specific values")
+    .option("--values <file>", "File with Platform-specific values")
```

Comment:
```suggestion
    .option("--values <file>", "File with Platform-specific values (TOML|YAML|JSON)")
```

I think it would be good to specify the expected file types